core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -192,6 +197,7 @@ angular.module('oppia').directive('questionsList', [
             ctrl.questionId = ctrl.question.getId();
             ctrl.questionStateData = ctrl.question.getStateData();
             ctrl.questionIsBeingUpdated = false;
+            ctrl.newQuestion = true;
```

Comment:
Just realized -- the naming of this seems off; it doesn't sound like a boolean (I would have expected it to be a question object).

Can you rename it so that it's clear that the variable stores a boolean (and the name describes what it stores)? Make sure not to start with a verb.

Reply:
Done